src/css/style.css

```diff
@@ -238,6 +238,11 @@ nav li a:visited {
   margin-bottom: 5px;
 }
 
+nav li a.on-page {
+  color: #92dce5;  
```

Comment:
ideally colors are always variables, at line `8` see the variables defined at `:root`. This also means all colors will be within the palette we chose a bit back.

Reply:
Thus yeah this color isn't in the palette so I feel like it doesn't quite match:
![2023-04-28_17-06](https://user-images.githubusercontent.com/8533113/235105567-32f32840-4e52-43c6-9e3a-e76941bc5f72.png)